rbac/server/db/users_query.py

```diff
@@ -80,6 +80,20 @@
         raise ApiNotFound("Not Found: No user with the id {} exists".format(next_id))
 
 
+async def delete_user_resource(conn, next_id):
+    """Database query to delete an individual user."""
+    resource = (
+        await r.table("users")
```

Comment:
I had a question on this, but as we've refined the "Delete a User" epic and created other necessary tickets around it. Now, I'm also okay with these changes.